install/includes/db-seed-base_url.rst

```diff
@@ -0,0 +1,6 @@
+.. tip::
+
+   **🤓 Preserve a restart ...**
+
+   You can set the base URL of your Zammad installation by setting the
+   ``ZAMMAD_HTTP_TYPE`` and ``ZAMMAD_FQDN`` environment variables.
```

Comment:
Without an example, I'm not sure how would an admin know what to set these variables to?

Perhaps we can document them like others in `appendix/configure-env-vars.rst` and then link from here?